torch/csrc/jit/python_arg_parser.cpp

```diff
@@ -0,0 +1,119 @@
+#include "python_arg_parser.h"
+
+namespace torch { namespace jit { namespace python {
+
+struct ParsedArgs {
+  std::vector<py::handle> vars;
+  std::string desc;
```

Comment:
It would be nice if you could add a comment saying what the format of this string is.